### PR TITLE
Update Swiftlint

### PR DESCRIPTION
**Pull request by pepicrft**
### Short description 📝
I'm updating SwiftLint to the most recent version.

### CODE REVIEW

File: Sources/ProjectDescription/AnalyzeAction.swift [hunk 1]
```diff
@@ -7,10 +7,6 @@ public struct AnalyzeAction: Equatable, Codable {
     /// Indicates the build configuration the product should be analyzed with.
     public let configuration: ConfigurationName
 
-    init(configuration: ConfigurationName) {
```
Comment (pepicrft) [inline Sources/ProjectDescription/AnalyzeAction.swift:10]: SwiftLint fixes are odd. I need to investigate them.

Comment (danyf90) [reply Sources/ProjectDescription/AnalyzeAction.swift:10]: I think default init are now authomatically synthesized

**Review by danyf90 — APPROVED** [review]

File: Sources/ProjectDescription/AnalyzeAction.swift (hunk 1 above)
Comment (pepicrft) [reply Sources/ProjectDescription/AnalyzeAction.swift:10]: Oh I see! That might be the issue then. I need to double check if the synthesized initializers will cause breaking changes.

Comment (fortmarek) [reply Sources/ProjectDescription/AnalyzeAction.swift:10]: As long as they don't include any custom defaults, they shouldn't. From what I can see, they should be fine imho

**Review by fortmarek — APPROVED** [review]

**Pull request merged**